src/rdkafka_int.h

```diff
@@ -957,6 +1002,17 @@ rd_kafka_fatal_error_code (rd_kafka_t *rk) {
 extern rd_atomic32_t rd_kafka_thread_cnt_curr;
 extern char RD_TLS rd_kafka_thread_name[64];
 
+#define wait_number_threads(num_threads, increment) do {        \
+                int32_t current_n_threads;              \
+                do { \
+                        current_n_threads = rd_atomic32_get(&rd_kafka_thread_cnt_curr); \
+                        while(current_n_threads + (increment) > (num_threads)) { \
+                                usleep(1000 * 100); \
```

Comment:
Should be configured?

Reply:
This macro is not used, so I will remove it.